actions/campaigns.go

```diff
@@ -154,7 +150,21 @@ func StartCampaign(c *gin.Context) {
 			"segment_ids": body.SegmentIDs,
 		}).WithError(err).Error("Unable to queue campaign for sending.")
 		c.JSON(http.StatusInternalServerError, gin.H{
-			"message": "Unable to publish campaign.",
+			"message": "Unable to start campaign.",
+		})
+		return
+	}
+
+	err = storage.UpdateCampaign(c, campaign)
+	if err != nil {
+		logger.From(c).WithFields(logrus.Fields{
+			"campaign_id": id,
+			"user_id":     u.ID,
```

Comment:
No need to add user_id in the **handler** function logs since it is already added by a middleware.